src/protocol/payload/tx.rs

```diff
@@ -785,4 +1125,37 @@ mod tests {
 
         assert_eq!(tx_v4, Tx::decode(&mut Cursor::new(&bytes)).unwrap());
     }
+
+    #[test]
+    #[ignore]
```

Comment:
why do we ignore this? also, it would be best to test with a non-empty tx, preferably from some original test vector

Reply:
We ignored all the unit tests so that we can run Ziggurat with just `cargo t`, though perhaps a feature flag would be more appropriate in the long run.

Reply:
> preferably from some original test vector
Agreed, I'll follow up with this if I can find some.